parsers/EIA.py

```diff
@@ -14,12 +14,13 @@
 
 import arrow
 from dateutil import parser, tz
-from requests import Session
-
 from parsers.ENTSOE import merge_production_outputs
 from parsers.lib.config import refetch_frequency
 from parsers.lib.utils import get_token
 from parsers.lib.validation import validate
+from requests import Session
+
+from electricitymap.contrib.libs.models.datapoints import ConsumptionBatch
```

Comment:
The imports using symlinks will break on windows, I suggest either moving the whole libs folder into contrib or importing it directly from libs instead. 

While it should not be an issue with our backend it might cause unexpected problems for our contributors if they try and run it locally.

Reply:
I removed the symlink and put it into contrib directly.